index.html

```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
                 </li>
             </ul>
         </div>
-    </div>
-
+        </div>
+    </section>
+    <div id="work">
+        <section id="tonic1">
+            <div class="card">
+                <a class="card_image" href="#"><img src="tonic1_image.svg" alt="tonic1"></a>
+                <div class="card_bottom">
+                    <h1 class="card_title">Tonic</h1>
+                    <ul class="card_ul">
+                        <li><p class="canopy">CANOPY</p></li>
+                        <li><img class="counter" src="Counter.svg" alt="counter"></li>
+                        <li><p class="text_li">Back End Dev</p></li>
+                        <li><img class="counter" src="Counter.svg" alt="counter"></li>
+                        <li><p class="text_li">2015</p></li>
+                    </ul>
+                    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li><p>HTML</p></li>
+                        <li><p>CSS</p></li>
+                        <li><p>JavaScript</p></li>
+                    </ul>
+                    <a class="see_project_button" href="#">See project</a>
+                </div>

```

Comment:
- [ ] Kindly remove the border color and try to match the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdessign_card](https://user-images.githubusercontent.com/70483835/227505173-d0f7fd89-5617-4cfe-952b-3cbf3c20923d.png) | ![figma_card](https://user-images.githubusercontent.com/70483835/227505164-a342752c-f8df-471e-90a4-564b26982edd.png)